integration-test/src/test/scala/com/cloudera/livy/test/JobApiIT.scala

```diff
@@ -76,7 +76,8 @@ class JobApiIT extends BaseIntegrationTestSuite with BeforeAndAfterAll with Logg
       assert(list.total === 1)
       val tempSessionId = list.sessions(0).id
 
-      livyClient.connectSession(tempSessionId).verifySessionIdle()
+      val s = livyClient.connectSession(tempSessionId)
```

Comment:
I thought users shouldn't submit RSC jobs while the session's starting.
I will fix the code to re-enable this use case then revert this change.